### PR TITLE
README and yml updates

### PR DESCRIPTION
**Pull request by e3betht**
Updating the readme with the custom config info and updating the yml file to include an example of the custom configurations

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 exclude_paths:
- - "/lib/testing.php"
- - "/vendor/"
+ - "/examples/*"
 engines:
   phpcodesniffer:
     enabled: true
+    config:
+      - custom_exclude_paths: "*/test.php","/vendor/"
```
Comment (GordonDiggs) [inline .codeclimate.yml]: Is there a benefit to listing these here instead of the global exclude paths?

File: engine.php [hunk 2]
```diff
@@ -58,7 +58,7 @@
                         'end' => $phpcs_issue_data['line']
                     )
                 ),
-                'remediation_points' => $phpcs_issue_data['severity'] * $phpcs_issue_data['severity'] * 100
+                'remediation_points' => $phpcs_issue_data['severity'] * 1.5 * 50000
```
Comment (GordonDiggs) [inline engine.php]: These points need to be emitted as integers

File: .codeclimate.yml (hunk 1 above)
Comment (e3betht) [inline .codeclimate.yml]: Yes - This allows you to exclude a file specifically from PHP_CodeSniffer but not from other analysis.

**Comment by GordonDiggs** [comment]
:+1: looks good

**Pull request merged**